openapi/p4pa-io-notification.openapi.yaml

```diff
@@ -108,34 +105,34 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/IoNotificationErrorDTO'
-
-  /service/{enteId}/{tipoDovutoId}:
-    get:
+  /send/message:
```

Comment:
thank you for the fix. Give a thought if we can improve the `/send/message` url too